index.ts

```diff
@@ -33,15 +34,17 @@ class ElectronForgeSslCodeSignPlugin extends PluginBase<ConfigTypes> {
 		forgeConfig: ResolvedForgeConfig,
 		makeResults: ForgeMakeResult[],
 	) => {
-		const squirrelMaker: IForgeResolvableMaker = forgeConfig.makers.find(
-			(maker) =>
-				(maker as IForgeResolvableMaker)?.name ===
-				"@electron-forge/maker-squirrel",
-		) as IForgeResolvableMaker
-
-		if (!squirrelMaker) {
+		const squirrelMaker: ForgeConfigMaker | undefined = forgeConfig.makers.find(
+			(maker) => "name" in maker && maker.name === "squirrel",
+		)
+		const squirrelResolvableMaker: ForgeConfigMaker | undefined =
+			forgeConfig.makers.find(
+				(maker) =>
+					"name" in maker && maker.name === "@electron-forge/maker-squirrel",
+			)
```

Comment:
squirrelResolvableMaker here resolves the object-based maker
squirrelMaker resolves the class-based maker